core/src/main/scala/frameless/CatalystNumeric.scala

```diff
@@ -16,4 +16,5 @@ object CatalystNumeric {
   implicit val framelessintNumeric       : CatalystNumeric[Int]        = of[Int]
   implicit val framelesslongNumeric      : CatalystNumeric[Long]       = of[Long]
   implicit val framelessshortNumeric     : CatalystNumeric[Short]      = of[Short]
+  // implicit val framelessfloatNumeric     : CatalystNumeric[Float]      = of[Float]
```

Comment:
I've noticed that Float is not added here, and scala's `scala.Float` actually is `scala.math.Numeric`. Shouldn't it be added here?

Reply:
Yes, good catch!